lessons/pydata/databases/index.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<p>Tady by se hodily nějaké příklady k zamyšlení na typy vazeb. Třeba novinový článek a autor (to je trochu chyták, článek může mít víc autorů).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>